packages/cli/src/commands/project/add.ts

```diff
@@ -32,12 +33,14 @@ export default async function add(
   const start = Date.now();
 
   const [name] = args;
+
   try {
-    await client.fetch('/projects', {
-      method: 'POST',
-      body: { name },
-    });
+    await createProject(client, { name });
```

Comment:
I see that the createProjects calls `/v1/projects` I'm not sure if this is different enough from `/projects`, may need to wait for someone from CLI to review as well.

Reply:
"If the e2e tests are happy then I'm happy" - @TooTallNate

😄 going to resolve this conversation